enterprise/models.py

```diff
@@ -445,6 +454,13 @@ class Meta:
         default=False,
     )
 
+    career_engagement_network_message = models.TextField(
```

Comment:
is this message same for every enterprise?

Reply:
how we will add this message? On django admin?

Reply:
yes